mkdocs.yml

```diff
@@ -68,33 +68,48 @@ plugins:
-    - 'Service Protection installation with Existing OCM': multicluster-gateway-controller/docs/installation/service-protection-installation.md
+    - 'Multi-Cluster':
+      - 'Control Plane installation with Existing OCM': multicluster-gateway-controller/docs/installation/control-plane-installation.md
+      - 'Service Protection installation with Existing OCM': multicluster-gateway-controller/docs/installation/service-protection-installation.md
+    - 'Single Cluster': kuadrant-operator/doc/development.md #need a new doc for this
   - 'Concepts and APIs':
-    - 'Gateways':
-      - 'Defining and Distributing Multicluster Gateways with OCM': multicluster-gateway-controller/docs/gateways/define-and-place-a-gateway.md
-      - 'Gateway Deletion': multicluster-gateway-controller/docs/gateways/gateway-deletion.md
     - 'DNSPolicy':
-      - 'Quickstart': multicluster-gateway-controller/docs/dnspolicy/dnspolicy-quickstart.md
-      - 'Reference': multicluster-gateway-controller/docs/dnspolicy/dns-policy.md
-      - 'DNS Health Checks': multicluster-gateway-controller/docs/dnspolicy/dns-health-checks.md
-      - 'DNS Providers': multicluster-gateway-controller/docs/dnspolicy/dns-provider.md
-      - 'ManagedZone Reference': multicluster-gateway-controller/docs/dnspolicy/managed-zone.md
+      - 'Overview': multicluster-gateway-controller/docs/dnspolicy/dns-policy.md
+      - 'Reference': multicluster-gateway-controller/docs/dnspolicy/dns-policy.md #need new doc
+    - 'ManagedZone':
+      -  'Overview': multicluster-gateway-controller/docs/dnspolicy/managed-zone.md          
+      -  'Reference': multicluster-gateway-controller/docs/dnspolicy/managed-zone.md #need new doc          
     - 'TLSPolicy':
-      - 'Reference': multicluster-gateway-controller/docs/tlspolicy/tls-policy.md
+      - 'Overview': multicluster-gateway-controller/docs/tlspolicy/tls-policy.md
+      - 'Reference': multicluster-gateway-controller/docs/tlspolicy/tls-policy.md #need new doc
     - 'AuthPolicy':
       - 'Overview': kuadrant-operator/doc/auth.md
       - 'Reference': kuadrant-operator/doc/reference/authpolicy.md
     - 'RateLimitPolicy':
       - 'Overview': kuadrant-operator/doc/rate-limiting.md
       - 'Reference': kuadrant-operator/doc/reference/ratelimitpolicy.md
   - 'How-to Guides':
-    - 'Multicluster Gateway Controller':
+    - 'Multicluster Gateways':
+      - 'Defining and Distributing Multicluster Gateways with OCM': multicluster-gateway-controller/docs/gateways/define-and-place-a-gateway.md
+      - 'Gateway Deletion': multicluster-gateway-controller/docs/gateways/gateway-deletion.md
       - 'Multicluster Gateways Walkthrough': multicluster-gateway-controller/docs/how-to/multicluster-gateways-walkthrough.md
-      - 'Metrics Walkthrough': multicluster-gateway-controller/docs/how-to/metrics-walkthrough.md
+    - 'Observability':
+      - 'Multi-Cluster Metrics Walkthrough': multicluster-gateway-controller/docs/how-to/metrics-walkthrough.md
```

Comment:
I think I'd make Observability the last one, after "Rate Limiting".

Can we also please bring _Authentication & Authorization > Observability_ into this group? (Like renamed "Authentication & Authorization Observability", I suppose.)